pkg/volumereplica/volumereplica.go

```diff
@@ -385,6 +387,45 @@ func buildVolumeCloneCommand(cStorVolumeReplica *cstor.CStorVolumeReplica, snapN
 	return cloneVolCmd
 }
 
+func SetTargetIp(fullVolName, targetIp string) error {
+	stdoutStderr, err := RunnerVar.RunCombinedOutput(VolumeReplicaOperator,
```

Comment:
This can be simplified using `zcmd` package. Reference: https://github.com/openebs/cstor-operators/blob/master/pkg/volumereplica/volumereplica.go#L669

Reply:
ok will do